internal/function/uast.go

```diff
@@ -20,14 +20,18 @@ import (
 )
 
 const (
-	uastCacheSize        = "GITBASE_UAST_CACHE_SIZE"
+	uastCacheSizeKey     = "GITBASE_UAST_CACHE_SIZE"
 	defaultUASTCacheSize = 10000
+
+	uastMaxBlobSizeKey     = "GITBASE_MAX_UAST_BLOB_SIZE"
+	defaultUASTMaxBlobSize = -1
 )
 
 var uastCache *lru.Cache
+var uastMaxBlobSize int
```

Comment:
we should specify _unit_, I think it make sense to have it in MB. But if we stick to bytes then `int` may be not good enough.

Reply:
`len(bytes)` is an int. You literally can't have a slice bigger than that, so no point in using any other type